docs/advanced_topics/api/v2/configuration.md

```diff
@@ -45,6 +45,23 @@ Wagtail provides three endpoint classes you can use:
 -   Documents {class}`wagtail.documents.api.v2.views.DocumentsAPIViewSet`
 
 You can subclass any of these endpoint classes to customize their functionality.
+
+For example:
+```python
+
+# In this case if you need to customize APIViewSet by setting a desired renderer class
```

Comment:
```suggestion
# In this case if you need to customize APIViewSet by setting a desired renderer class
```

Let's move this line above and also use `customise` not `customize` as we prefer British spelling in docs.

Otherwise it looks great. I will make this change and get this merged in.